pyscriptjs/src/main.ts

```diff
@@ -204,9 +207,34 @@ export class PyScriptApp {
         // XXX: maybe the following calls could be parallelized, instead of
         // await()ing immediately. For now I'm using await to be 100%
         // compatible with the old behavior.
+        logger.info('importing pyscript');
+
+        // Save and load pyscript.py from FS
+        runtime.interpreter.FS.writeFile("pyscript.py", pyscript, { encoding: "utf8" });
+
+        // add `create_custom_element` and the current `PyScriptApp` to the Python namespace
+        // in the pyscript module scope
+        runtime.globals.set('create_custom_element', create_custom_element);
+        runtime.globals.set('_pyscript_app', this);
+
+        // inject it into the PyScript module scope
+        // TODO: Currently adding the imports for backwards compatibility, we should
+        //       probably remove it
+        await runtime.run(`
+        import pyscript
+        pyscript.create_custom_element = create_custom_element
+        pyscript._pyscript_app = _pyscript_app
+        pyscript.PyScript.set_version_info('${version}')
```

Comment:
You could assign these attributes onto the Pyscript module without mucking with the Python global scope at all:
```js
let pyscript = runtime.pyimport("pyscript");
pyscript.create_custom_element = create_custom_element
pyscript._pyscript_app = this;
pyscript.PyScript.set_version_info(version);
pyscript.destroy();
```
generally I try to avoid string interpolating into code I'm going to exec when possible.